go.sum

```diff
@@ -1,35 +1,93 @@
+bazil.org/fuse v0.0.0-20160811212531-371fbbdaa898/go.mod h1:Xbm+BRKSBEpa4q4hTSxohYNQpsxXPbPry4JJWOB3LB8=
```

Comment:
Merging with master had conflicts at `go.mod` and `go.sum`,  running `go mod tidy` resulted in these changes